helper/hostutil/hostinfo_util.go

```diff
@@ -1,51 +1,97 @@
 package hostutil
 
-import (
-	"github.com/shirou/gopsutil/v3/host"
-	"github.com/shirou/gopsutil/v3/mem"
-)
-
-// VirutalMemoryStat holds commonly used memory measurements. We must have a
+// VirtualMemoryStat holds commonly used memory measurements. We must have a
 // local type here in order to avoid building the gopsutil library on certain
 // arch types.
+//
+// This struct is copied from https://github.com/shirou/gopsutil/blob/b49f37e9f30f49530cf2ad6038a4dac1b746c8f7/mem/mem.go#L15
```

Comment:
I believe copying the code means we will also need to copy the license https://github.com/shirou/gopsutil/blob/master/LICENSE

Reply:
Thanks for catching this! I've done so in 1d40ffb. Lmk if that looks okay.